### PR TITLE
issue-49 Contact Block simplified

### LINKED ISSUE #49: Simplify "Contact" block's markup
As seen in https://www.zeiss.de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/europaeischer-erfinderpreis-2018.html#presseinformation

image

This is handeled by the "Contact" block. This task is logged to simplify the markup generated by this block here
https://github.com/hlxsites/zeiss/blob/b61a426d949efe128cb53fc54522df7a8a1568cc/blocks/contact/contact.js#L5-L74

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #49 

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.live/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020
- After: https://issue-49--zeiss--hlxsites.hlx.live/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020

### CODE REVIEW

**Comment by jindaliiita** [comment]
Will write test cases

**Review by alinarublea — APPROVED** [review]

**Review by sdmcraft — COMMENTED** [review]
The contact block overlaps with the block above it [on this page ](https://issue-49--zeiss--hlxsites.hlx.page/drafts/satyam/sample-template). You may want to check why that's happening.

image

**Comment by jindaliiita** [comment]
> 

@bosschaert Style https://github.com/hlxsites/zeiss/blob/main/blocks/cards/cards.css#L57 is causing issue for cards-container on following page
https://main--zeiss--hlxsites.hlx.page/drafts/satyam/sample-template
Can you please check it.
cc @sdmcraft

File: blocks/contact/contact.js [hunk 1]
```diff
@@ -1,83 +1,103 @@
...
+}
+
+function template(block, items, placeholders) {
+  const div = document.createElement('div');
+
+  const profileDetails = document.createElement('div');
+  profileDetails.classList.add('profile-collection__settings', 'grid__container', `primary-count-${items.length}`);
+
+  const innerProfileDetailsDiv = document.createElement('div');
+  innerProfileDetailsDiv.classList.add('grid__structure', 'profile-collection__structure', 'profile-collection__structure--primary');
```
Comment (sdmcraft) [inline blocks/contact/contact.js:82]: Is it possible to get rid of these 3 nested divs as well and directly put the markup inside the `block` div?

**Comment by sdmcraft** [comment]
> > 
> 
> @bosschaert Style https://github.com/hlxsites/zeiss/blob/main/blocks/cards/cards.css#L57 is causing issue for cards-container on following page https://main--zeiss--hlxsites.hlx.page/drafts/satyam/sample-template

Thanks for checking. Looks like this is related to `cards` block so can be tracked separately.

**Comment by bosschaert** [comment]
Thanks @jindaliiita I created #215 for it.

File: blocks/contact/contact.js (hunk 1 above)
Comment (jindaliiita) [reply blocks/contact/contact.js:82]: Will check and is possible will remove this

Comment (sdmcraft) [reply blocks/contact/contact.js:82]: Thanks. Let me also explain my reasoning behind this.
If we remove these nested divs and also the exception for `contact` block [from here](https://github.com/hlxsites/zeiss/blob/ea20c954e14e4de68e28ef1f20dba78ee5b6e677/styles/simplified-styles.css#L151-L159) then the block itself could provide the grid layout. The advantage would be that this article-list block would inherit the global styles applicable for all the blocks thus providing a consistent styling for all the blocks in terms of margins, paddings etc.

**Pull request merged**